src/core/translations/dutch-translations.constant.js

```diff
@@ -13,6 +13,7 @@ angular.module('udb.core')
   {
     'BE': 'België',
     'NL': 'Nederland',
+    'ZZ': 'Locatie in overleg met de school',
```

Comment:
I keep thinking this is weird copy. (Mostly the "in overleg __met de school__")
If this is only intended for about ±300 events, will the organizers of other events know what this means?
Wouldn't "Locatie in overleg met de organisator" or something along those lines be better?

Reply:
(To be discussed with Corneel / Stan)

Reply:
They understand your concern, however they already put a lot of thought in it and this copy seems like the best solution...